thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -196,6 +196,29 @@ public Builder setInstallationID(
             return this;
         }
 
+        /**
+         * Instantiate new Builder without Context.
+         * This method is for internal use only. Do not call it from your application.
+         *
+         * @param app Kii Cloud Application.
+         * @param owner Specify who uses the ThingIFAPI.
+         * @return Builder instance.
+         */
+        @NonNull
+        public static Builder _newBuilder(
```

Comment:
Since this is not scope of this PR, I added it to TODO list in issue #281.  I will consider it during implementation in another PR.